deploy/local/Makefile

```
@@ -98,58 +127,98 @@ annotation: ## 🏷️ Start Annotation UI (Gradio)
 stop: ## 🛑 Stop all services
 	@echo "🛑 Stopping all services..."
 	@echo "🔧 Using: $(COMPOSE_CMD)"
+	@$(MAKE) -s stop-aap-mock
 	@$(MAKE) -s stop-loki-stack
 	@$(MAKE) -s stop-postgres
 	@$(MAKE) -s stop-phoenix
-	@$(MAKE) -s stop-embedding
 	@$(MAKE) -s kill-ports
 	@echo "👋 All services stopped"
 
 stop-postgres: ## 🛑 Stop PostgreSQL database
-	@$(COMPOSE_CMD) -f compose.yaml down postgres
+	@$(COMPOSE_CMD) -f compose.yaml stop postgres || true
+	@$(COMPOSE_CMD) -f compose.yaml rm -f postgres || true
 	@echo "   ✓ PostgreSQL stopped"
 
 stop-phoenix: ## 🛑 Stop Phoenix
-	@$(COMPOSE_CMD) -f compose.yaml down phoenix
+	@$(COMPOSE_CMD) -f compose.yaml stop phoenix || true
+	@$(COMPOSE_CMD) -f compose.yaml rm -f phoenix || true
 	@echo "   ✓ Phoenix stopped"
 
 stop-loki-stack: ## 🛑 Stop Loki stack (Loki, Loki MCP Server, Grafana, Promtail)
-	@$(COMPOSE_CMD) -f compose.yaml down loki-mcp-server grafana promtail loki
+	@$(COMPOSE_CMD) -f compose.yaml stop loki-mcp-server grafana promtail loki || true
+	@$(COMPOSE_CMD) -f compose.yaml rm -f loki-mcp-server grafana promtail loki || true
 	@echo "   ✓ Loki stack stopped"
 
-stop-embedding: ## 🛑 Stop Embedding Service
-	@$(COMPOSE_CMD) -f compose.yaml down alm-embedding
-	@echo "   ✓ Embedding service stopped"
+stop-aap-mock: ## 🛑 Stop AAP Mock
+	@$(COMPOSE_CMD) -f compose.yaml stop aap-mock || true
+	@$(COMPOSE_CMD) -f compose.yaml rm -f aap-mock || true
+	@echo "   ✓ AAP Mock stopped"
```

```suggestion
stop-postgres: ## 🛑 Stop and remove PostgreSQL database
	@$(COMPOSE_CMD) -f compose.yaml down postgres || true
	@echo "   ✓ PostgreSQL removed"

stop-phoenix: ## 🛑 Stop and remove Phoenix
	@$(COMPOSE_CMD) -f compose.yaml down phoenix || true
	@echo "   ✓ Phoenix removed"

stop-loki-stack: ## 🛑 Stop and remove Loki stack (Loki, Loki MCP Server, Grafana, Promtail)
	@$(COMPOSE_CMD) -f compose.yaml down loki-mcp-server grafana promtail loki || true
	@echo "   ✓ Loki stack removed"

stop-aap-mock: ## 🛑 Stop and remove AAP Mock
	@$(COMPOSE_CMD) -f compose.yaml down aap-mock || true
	@echo "   ✓ AAP Mock removed"
```

@trujillm  can you pls update to `down` to support both podman-compose and docker compose